### ISSUE TITLE
Spinner border is clipped

### ISSUE THREAD

**Issue by RenzHoly** [palantir/blueprint#2966]
Spinner border is clipped when spinner is small
#### Environment

- __Package version(s)__: 
3.6.1
- __Browser and OS versions__: 
MacOS 10.13, Electron 3.0.0
#### Steps to reproduce
```jsx
<div
  style={{
  display: 'inline-block',
  verticalAlign: 'middle',
}}>
  <Spinner
    size={16}
    intent={Intent.PRIMARY}
    value={this.props.progress}
  />
</div>
```
#### Actual behavior

![image](https://user-images.githubusercontent.com/8287771/45917518-81226480-beab-11e8-9d54-2ddcf84d645e.png)
![image](https://user-images.githubusercontent.com/8287771/45917520-82ec2800-beab-11e8-864b-26d408163bf8.png)
![image](https://user-images.githubusercontent.com/8287771/45917521-841d5500-beab-11e8-81cb-0a7ceb732761.png)

#### Expected behavior
@blueprintjs/core@3.2.0 is ok
![image](https://user-images.githubusercontent.com/8287771/45917529-c6469680-beab-11e8-93d5-191296a7d8be.png)

#### Possible solution
downgrade to @blueprintjs/core@3.2.0

**Comment by giladgray** [comment]
@RenzHoly I cannot reproduce this on the docs site. Please confirm that you do not have any `overflow` styles, as that will unavoidably trim the overflow and produce these artifacts.

fwiw, downgrading is not a "solution" as it locks you off from any actual fixes. an appropriate solution here is to audit your app styles for properties that would affect a component's `overflow`.

**Comment by RenzHoly** [comment]
@giladgray I can reproduce this on the docs site, 
![image](https://user-images.githubusercontent.com/8287771/46013429-e3bf6e80-c0fe-11e8-93b5-d14e17a7748b.png)

**Comment by giladgray** [comment]
oh yeah that's a repro! i will look into this.

**Comment by giladgray** [comment]
@RenzHoly i just opened a sweet fix at #2970

**Comment by RenzHoly** [comment]
@giladgray good job! thanks a lot
